keep-ui/app/(keep)/providers/provider-form.tsx

```diff
@@ -1111,4 +851,446 @@ const ProviderForm = ({
   );
 };
 
+function GroupFields({
```

Comment:
Let's move the UI components such as GroupFields, FormField to `/ui/` folder

Reply:
Instead of moving them to some top-level `/ui/` folder, I created a `form-fields` tsx file inside the `providers` folder. I did this primarily because the fields are very "provider form" specific based on their prop types. They'll most likely not be consumed or required by any other component outside the `ProviderForm`.